### PR TITLE
A little extra test coverage for Service Interface Inspect.

### PR DESCRIPTION
**Pull request by mgoulish**
Every little bit of coverage helps.
And this is a *very* little bit.

### CODE REVIEW

File: client/van_serviceinterface_inspect_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+		} else {
+			assert.Check(t, err, "Creation failed.")
+
+			// When we inspect the VanServiceInterface, make sure that the
+			// expected values have been set.
+			serviceInterface, err := cli.VanServiceInterfaceInspect(ctx, testcase.addr)
+			assert.Check(t, err, "Inspectionion failed.")
+
+			assert.Equal(t, testcase.addr, serviceInterface.Address, "Error in address.")
+			assert.Equal(t, testcase.proto, serviceInterface.Protocol, "Error in protocol.")
```
Comment (grs) [inline client/van_serviceinterface_inspect_test.go:105]: assert on port also?

File: client/van_serviceinterface_inspect_test.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+				"\n\nTest %s failure: The expected error |%s| was not reported.\n",
+				testcase.namespace,
+				testcase.expectedCreationError)
+		} else {
+			assert.Check(t, err, "Creation failed.")
+
+			// When we inspect the VanServiceInterface, make sure that the
+			// expected values have been set.
+			serviceInterface, err := cli.VanServiceInterfaceInspect(ctx, testcase.addr)
+			assert.Check(t, err, "Inspectionion failed.")
```
Comment (grs) [inline client/van_serviceinterface_inspect_test.go:102]: typo: inspecti-onion

File: client/van_serviceinterface_inspect_test.go [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+			assert.Equal(t, testcase.addr, serviceInterface.Address, "Error in address.")
```
Comment (grs) [inline client/van_serviceinterface_inspect_test.go:104]: Would be nice to have a slightly more informative message, e.g. include the test case and show the expected and actual values?

Comment (mgoulish) [reply client/van_serviceinterface_inspect_test.go]: Yes that would be nice.  I started to do that and then my brains all went away.
All fixed now.   (The code. Not my brains.)

Also, I took the 'onion' out of inspectionion.

**Comment by grs** [comment]
The test error here is unrelated, it seems something is not right with the console build at present. 

```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@patternfly%2freact-icons - Not found
npm ERR! 404 
npm ERR! 404  '@patternfly/react-icons@^4.4.2' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'gilligan-master'
```

**Pull request merged**